scipy/signal/signaltools.py

```diff
@@ -33,7 +33,8 @@
     from fractions import gcd
 
 
-__all__ = ['correlate', 'fftconvolve', 'convolve', 'convolve2d', 'correlate2d',
+__all__ = ['correlate', 'correlate2d',
+           'convolve', 'convolve2d', 'fftconvolve', 'oaconvolve',
```

Comment:
CircleCI complains because of ref problems, probably because you need to add this to `scipy/signal/__init__.py` itself as an import, and in the top docstring in the `Convolution` section

Reply:
I think this is fixed

Reply:
I see no changes to `scipy/signal/__init__.py` in the diff, did you forget to push?

Reply:
I just pushed it

Reply:
Travis seems okay now.